src/SessionSetup/OtherUserDialog.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "SessionSetup/OtherUserDialog.h"
+
+#include <QSettings>
+#include <memory>
+
+#include "ui_OtherUserDialog.h"
+
+namespace {
+constexpr const char* kRememberKey = "OtherUserDialog.RememberKey";
```

Comment:
Nit: The anonymous namespace is not needed with a constexpr (or const) variable. Internal linkage is implied automatically.

Reply:
true. I forgot. Thanks. done!